packages/daisy/src/components/card/card-image.tsx

```diff
@@ -0,0 +1,14 @@
+import { component$, HTMLAttributes, QwikIntrinsicElements, Slot } from '@builder.io/qwik';
+import {
+  CardImage as HeadlessCardImage,
+} from '@qwik-ui/headless';
+export type HTMLImgProps = QwikIntrinsicElements['img'];
+
+type CardImageProps = HTMLAttributes<HTMLElement> & HTMLImgProps;
```

Comment:
🔥 That's so cool. I was looking for a type-safe way to provide native HTML overrides to our components.